### PR TITLE
Issue #3718: Add varargs support for NoWhitespaceBefore

### PR DESCRIPTION
**Pull request by robertpainsi**
Issue https://github.com/checkstyle/checkstyle/issues/3718

### CODE REVIEW

**Comment by rnveach** [comment]
wercker failure is because `Orekit` project uses whitespaces before the `...`.
> [ERROR] src/main/java/org/orekit/models/earth/tessellation/EllipsoidTessellator.java:[682,75] (whitespace) NoWhitespaceBefore: '...' is preceded with whitespace.

https://github.com/CS-SI/Orekit/blob/66dd34b3742cb540fd88dcdf0cc052ad23f0b657/src/main/java/org/orekit/models/earth/tessellation/EllipsoidTessellator.java#L682

**Comment by robertpainsi** [comment]
> wercker failure is because Orekit project uses whitespaces before the ....

@rnveach, do I've to fix that?

**Comment by romani** [comment]
We need to contact that project and find out is such formatting on purpose or it is just mistake.

**Comment by romani** [comment]
looks like they moved from github to https://www.orekit.org/forge/projects/orekit/repository, I requested account on their web site to contact engineers of this project.

**Comment by romani** [comment]
I got a reply from Orekit project:

> It was on purpose, but was mainly a bad habit. I think
it should be better to fix it

It is ok to keep that token as default.
@robertpainsi , please prepare PR to Orekit to fix violation that will resolve CI and we can merge PR.
thanks for cooperation.

> The forge account is in fact needed only to register issues in the
bug tracker. If you want to send a mail to the orekit-developers
list, you will have to register there
<https://www.orekit.org/wws/subscribe/orekit-developers>,
this registration is automatic, there are no approval required.

in future contact with Orekit developers should as described above.

File: src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/NoWhitespaceBeforeCheck.java [hunk 1]
```diff
@@ -35,9 +35,9 @@
  * <p> By default the check will check the following operators:
  *  {@link TokenTypes#SEMI SEMI},
  *  {@link TokenTypes#POST_DEC POST_DEC},
- *  {@link TokenTypes#POST_INC POST_INC}.
- * {@link TokenTypes#DOT DOT} is also an acceptable token in a configuration
- * of this check.
+ *  {@link TokenTypes#POST_INC POST_INC},
+ *  {@link TokenTypes#DOT DOT},
+ *  {@link TokenTypes#ELLIPSIS ELLIPSIS}.
```
Comment (romani) [inline src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/NoWhitespaceBeforeCheck.java]: why DOT is become mentioned in documentation as default ?

Comment (rnveach) [inline src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/NoWhitespaceBeforeCheck.java]: Looks like `COMMA` is missing too.

Comment (robertpainsi) [inline src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/NoWhitespaceBeforeCheck.java]: Fixed and updated.

**Comment by romani** [comment]
@robertpainsi , changes are fine, as soon all CIs pass we could merge. Thanks a lot for help !

**Comment by robertpainsi** [comment]
Thanks guys! Always a pleasure to work with you :smile:

**Pull request merged**